src/FSAL/FSAL_CORTXFS/handle.c

```diff
@@ -1486,16 +1488,18 @@ static  inline fsal_status_t kvsfs_getattrs(struct fsal_obj_handle *obj_hdl,
 
 	myself =
 		container_of(obj_hdl, struct kvsfs_fsal_obj_handle, obj_handle);
+	retval = cfs_fh_from_ino(myself->cfs_fs, &ino, &fh);
```

Comment:
Seems like this is temporary changes where we are getting the FH from disk. And it is because still we aren't using FH at all the place? 
If so just add a comment that we can get rid of this call and directly get the FH from `kvsfs_fsal_obj_handle`
whenever it will be right thing to do, can be done at all the other places too.

Reply:
Yes, will add the discussed comment and address them in [EOS-15256](https://jts.seagate.com/browse/EOS-15256) and [EOS-15257](https://jts.seagate.com/browse/EOS-15256).

Reply:
I am not sure why this change was removed and now we are getting FH from KVSFS FSAL Object. I believe FH inside KSVFS FSAL Object will not be update all the time. So the change which was earlier was fine. If we are facing problem because of that than that needs to be debug further. If earlier change works fine, please add the comment as suggested in first time.